### PR TITLE
Enhance/5098 fix setup snippet toggle UI

### PR DESCRIPTION
**Pull request by kuasha420**
## Summary

Addresses issue:

- #5098

## PR Author Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [x] Run the code.
- [x] Ensure the acceptance criteria are satisfied.
- [x] Reassess the implementation with the IB.
- [x] Ensure no unrelated changes are included.
- [x] Ensure CI checks pass.
- [x] Check Storybook where applicable.
- [x] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [x] Ensure the PR has the correct target branch.
- [x] Double-check that the PR is okay to be merged.
- [x] Ensure the corresponding issue has a ZenHub release assigned.
- [x] Add a changelog message to the issue.

### CODE REVIEW

File: assets/sass/components/global/_googlesitekit-use-snippet-switch.scss [hunk 1]
```diff
@@ -0,0 +1,36 @@
+/**
+ * UseSnippetSwitch styles.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
```
Comment (techanvil) [inline assets/sass/components/global/_googlesitekit-use-snippet-switch.scss:4]: Ah, this old copy/paste chestnut :)
```suggestion
 * Site Kit by Google, Copyright 2022 Google LLC
```

File: assets/js/modules/tagmanager/components/common/SettingsUseSnippetSwitch.js [hunk 2]
```diff
@@ -34,12 +34,19 @@ export default function SettingsUseSnippetSwitch() {
...
+						'google-site-kit'
+				  )
+				: __(
+						'Site Kit will not add the code to your site.',
+						'google-site-kit'
+				  ) }
+		</p>
+	);
 
 	return <UseSnippetSwitch description={ description } />;
```
Comment (techanvil) [inline assets/js/modules/tagmanager/components/common/SettingsUseSnippetSwitch.js:51]: Not that there's anything wrong with the above change, but I'm wondering why not make the same change as in [assets/js/modules/analytics-4/components/common/SettingsUseSnippetSwitch.js](https://github.com/google/site-kit-wp/pull/5115/files#diff-44b7a10d2c41b0536f8423800377958ab5cdfd9b7f24d096a530c6baa4440a7f), for the sake of consistency? I.e.,

```js
return <UseSnippetSwitch description={ <p>{ description }</p> } />;
```

**Review by techanvil — CHANGES_REQUESTED** [review]
Hey @kuasha420, looks great, just a couple of trivial comments from me. Cheers

File: assets/sass/components/global/_googlesitekit-use-snippet-switch.scss (hunk 1 above)
Comment (kuasha420) [reply assets/sass/components/global/_googlesitekit-use-snippet-switch.scss]: Only if we could use current_year in the comments.

File: assets/js/modules/tagmanager/components/common/SettingsUseSnippetSwitch.js (hunk 2 above)
Comment (kuasha420) [reply assets/js/modules/tagmanager/components/common/SettingsUseSnippetSwitch.js]: Good catch, fixed.

**Review by techanvil — COMMENTED** [review]
LGTM, nice one @kuasha420!

File: assets/sass/components/global/_googlesitekit-use-snippet-switch.scss (hunk 1 above)
Comment (techanvil) [reply assets/sass/components/global/_googlesitekit-use-snippet-switch.scss]: Hahah, yes that would be handy. I keep meaning to setup a VSCode snippet for this, which could insert the current year along with the rest of the header...

**Pull request merged**